app/templates/subsidy/applications/edit.hbs

```diff
@@ -17,17 +17,37 @@
   </Group>
   <Group class="au-c-toolbar__group--col">
     <AuButtonGroup>
+      {{#if this.canDownload}}
+        <AuDropdown
+          @title="Bestanden Downloaden"
+          @buttonLabel="Account settings"
+          @alignment="right"
+          @skin="secondary"
+          @icon="download"
+          @iconAlignment="left"
+          role="menu"
+          class="au-u-hide-on-print"
+        >
+          <AuButton @skin="link" role="menuitem" {{on "click" this.exportSubsidyAsPDF}}>
+            Download huidige stap (PDF)
+          </AuButton>
+          <AuButton @skin="link" role="menuitem" {{on "click" this.downloadBijlagen}}>
```

Comment:
Another strange thing I noticed. The button is visible even if there are no download links, so if the user clicks it nothing happens.

Probably an edge case, and fixing it might be tricky since it depends on the form being rendered first so maybe not worth putting time in it?

Reply:
Good idea! Just pushed a simple check for it